package.json

```diff
@@ -16,11 +16,13 @@
     "metro-config": "^0.66.2",
     "react": "17.0.2",
     "react-native": "0.66.3",
+    "react-native-pager-view": "^5.4.9",
```

Comment:
`react-native-pager-view` had to be installed because of `react-native-tab-view`.
https://github.com/satya164/react-native-tab-view#installation

This probably could be replaced with **Tab Navigation** for more convenience.